content/id/docs/concepts/services-networking/service-topology.md

```diff
@@ -186,5 +186,5 @@ spec:
 
 
 * Baca tentang [mengaktifkan topologi Service](/docs/tasks/administer-cluster/enabling-service-topology)
-* Baca [menghubungkan aplikasi dengan Service](/id/docs/concepts/services-networking/connect-applications-service/)
+* Baca [menghubungkan aplikasi dengan Service](/id/docs/tutorials/services/connect-applications-service/)
```

Comment:
@Arhell This directory does not exists atm. This PR https://github.com/kubernetes/website/pull/36669 needs to be applied first before we can continue.

Reply:
Ok thanks, I'll close for now